packages/app-builder/src/routes/ressources+/scenarios+/$scenarioId+/testrun+/create.tsx

```diff
@@ -89,10 +91,21 @@ export function CreateTestRun({
 
   return (
     <ModalV2.Root open={open} setOpen={setOpen}>
-      <ModalV2.Trigger
-        render={children}
-        disabled={!hydrated || !shouldAllowCreate}
-      />
+      {shouldAllowCreate ? (
+        <ModalV2.Trigger render={children} disabled={!hydrated} />
+      ) : (
+        <Tooltip.Default content={t('scenarios:testrun.not_allowed')}>
+          <Button
+            disabled
+            variant="primary"
+            className="isolate h-10 w-fit cursor-not-allowed"
+          >
+            <Icon icon="plus" className="size-6" aria-hidden />
+            {t('scenarios:create_testrun.title')}
+          </Button>
+        </Tooltip.Default>
+      )}
```

Comment:
Lift up the conditional :
1. If allowed return the modal
2. If not return the disabled button with tooltip only